base_location_nuts/wizard/nuts_import_view.xml

```diff
@@ -36,7 +35,7 @@
 <menuitem action="nuts_import_action"
           id="nuts_import_menu"
           name="Import NUTS 2013"
-          parent="base.menu_localisation"
+          parent="sales_team.menu_localisation"
```

Comment:
Right, @pedrobaeza , but this module doesn't depend on that one :confused: It only depended on base before...